hemco_interface.F90

```diff
@@ -367,9 +363,9 @@ subroutine HCOI_Chunk_Init()
             write(iulog,*) "https://doi.org/10.5194/gmd-14-5487-2021 (Lin et al., 2021)"
             write(iulog,*) "HEMCO_CESM interface version 2.0.0"
```

Comment:
The interface version number has to be updated here. Perhaps this could be stored somewhere more prominently to make it easier to maintain.

Reply:
I will remove this and add DOI to the GitHub Readme.

Reply:
An example of the doi badge for the release is here: https://github.com/geoschem/gchp.

Reply:
Thanks Lizzie!